examples/tutorial/04-mm1-features.mm1

```diff
@@ -94,9 +114,21 @@ do (def (conj-prove refine t)
   )
   @ refine t (f lhs));
 
+-- Examples of using `conj-prove`
+
 theorem _: $ a /\ b /\ c /\ (d /\ e) -> d $ = conj-prove;
 theorem _: $ a /\ b /\ (d /\ e) -> d $ = conj-prove;
 theorem _: $ a /\ b /\ (e /\ d) -> d $ = conj-prove;
 theorem _: $ d /\ b /\ (e /\ d) -> d $ = conj-prove;
 theorem _: $ d /\ b /\ (e /\ d) -> b $ = conj-prove;
 theorem _: $ b -> b $ = conj-prove;
+
+-- Call `conj-prove` with `dbg` in place of `refine` to see the script that
+-- it produces
+do (conj-prove dbg $ a /\ b /\ (e /\ d) /\ f /\ g -> d $);
```

Comment:
That's clever, I've actually never tried to do it this way. Another way to see the proof of any theorem is the `get-proof` function, which shows the fully elaborated proof term after `refine` has done its job. For example:
```suggestion
do (conj-prove dbg $ a /\ b /\ (e /\ d) /\ f /\ g -> d $);

-- you have to give it a name to call `get-proof`
theorem foo: $ a /\ b /\ (e /\ d) /\ f /\ g -> d $ = conj-prove;
do {
  -- borrowed from peano.mm1
  (def (get-proof x) (nth 1 @ @ nth 6 @ get-decl x))
  
  (display @ pp @ get-proof 'foo)
  -- (anwl (a /\ b /\ (e /\ d) /\ f) g d
  --   (anwl (a /\ b /\ (e /\ d)) f d
  --     (anwr (a /\ b) (e /\ d) d (anwr e d d (id d)))))
};
```